models/articles.model.js

```diff
@@ -1,5 +1,26 @@
 const db = require("../db/connection");
 
+exports.selectAllArticles = () => {
+  return db
+    .query(
+      `SELECT articles.author, articles.title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.comment_id) AS comment_count
+      FROM articles 
+    JOIN comments ON articles.article_id = comments.article_id
```

Comment:
you're going to need a left join here. This is because if there are any article_id's without a comment they won't be included in the table with just a join 👀